app/dao/statistics_dao.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime, timedelta
+
+from flask import current_app
+from sqlalchemy.exc import IntegrityError, SQLAlchemyError
+
+from app import db
+from app.dao.dao_utils import transactional
+from app.models import (
+    JobStatistics,
+    EMAIL_TYPE,
+    SMS_TYPE,
+    LETTER_TYPE,
+    NOTIFICATION_STATUS_TYPES_FAILED,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_SENT)
+from app.statsd_decorators import statsd
+
+
+@transactional
+def timeout_job_counts(notifications_type, timeout_start):
```

Comment:
there are, just there was some aliasing going on. The aliasing is removed so tests should be easier to find.